src/components/workflowEditor/EmptyWorkflow.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { Component } from 'react'
+import { EmptyWorkflowProps, EmptyWorkflowState } from './types'
+import { DialogForm, DialogFormSubmit, ServerErrors, showError } from '@devtron-labs/devtron-fe-common-lib'
+import { createWorkflow, updateWorkflow } from './service'
+import { toast } from 'react-toastify'
+import error from '../../assets/icons/misc/errorInfo.svg'
+
+export default class EmptyWorkflow extends Component<EmptyWorkflowProps, EmptyWorkflowState> {
+    _inputName: HTMLInputElement
+
+    constructor(props) {
+        super(props)
+        this.state = {
+            name: '',
+            showError: false,
+            loading: false,
+        }
+    }
+
+    componentDidMount() {
+        if (this._inputName) {
+            this._inputName.focus()
+        }
+    }
+
+    handleWorkflowName = (event): void => {
+        this.setState({ name: event.target.value })
+    }
+
+    saveWorkflow = (event): void => {
+        event.preventDefault()
+        this.setState({ showError: true, loading: true })
+        let request = {
+            appId: +this.props.match.params.appId,
+            name: this.state.name,
+        }
+        if (!this.isNameValid()) return
+        let message = 'Empty Workflow Created successfully'
+        let promise = createWorkflow(request)
+        promise
+            .then((response) => {
+                toast.success(message)
+                this.setState({
+                    name: response.result.name,
+                    showError: false,
+                    loading: false,
+                })
+                this.props.onClose()
+                this.props.getWorkflows()
+            })
+            .catch((error: ServerErrors) => {
+                showError(error)
+                this.setState({
+                    loading: false,
+                })
+                this.props.onClose()
+            })
+    }
+
+    isNameValid(): boolean {
+        return !!this.state.name?.length
+    }
+
+    render() {
+        let isValid = this.isNameValid()
+        return (
+            <DialogForm
+                title={'Create job workflow'}
+                className=""
+                close={(event) => this.props.onClose()}
+                onSave={this.saveWorkflow}
+                isLoading={this.state.loading}
+                closeOnESC={true}
+            >
+                <label className="form__row">
+                    <span className="form__label dc__required-field">Workflow Name</span>
+                    <input
+                        autoComplete="off"
```

Comment:
We should map the label and input using id and remove the empty classname on Dialog form

Reply:
done